README.md

```diff
@@ -15,25 +15,43 @@ These benchmarks can be executed on a vanilla Kubernetes or OpenShift cluster. I
+Different scenarios can be customized under `config/benchmarks/scenarios/benchmarks`. Current the benchmarks support two testing scenarios:
+
+* Ingestion path scenarios: [suppored configuration](https://github.com/observatorium/loki-benchmarks/blob/1a0a9e8f6190475b6c1bfacb5a31a88bd76cbb36/internal/config/config.go#L76-L81), this test will generate X amount of logs throughout a 30 minute window that's supposed to represent a full day of log ingestion.
+* Query path scenarios: [supported configuration](https://github.com/observatorium/loki-benchmarks/blob/1a0a9e8f6190475b6c1bfacb5a31a88bd76cbb36/internal/config/config.go#L102-L108), the theory behind this test is to generate the amount of data that would be queried before it starts running the queries.
 
 ## Running Benchmarks
 
-Use the `make run-rhobs-benchmarks` or `make run-operator-benchmarks` to execute the benchmark program with the RHOBS or operator deployment styles on OpenShift respectively. Upon successful completion, a JSON and XML file will be created in the `reports/date+time` directory with the results of the tests.
+### Prerequisites
 
-## Troubleshooting
+The `run-operator-benchmarks` expects the following two env vars to be set `LOKI_OPERATOR_REGISTRY` `LOKI_STORAGE_BUCKET`.
+E.g
+
+```shell
+export LOKI_OPERATOR_REGISTRY=jmarcal
+export LOKI_STORAGE_BUCKET=jmarcal-loki-benchmark-storage
+```
 
-During benchmark execution, use [hack/scripts/ocp-deploy-grafana.sh](hack/scripts/ocp-deploy-grafana.sh) to deploy grafna and connect to Loki as a datasource: 
-- Use a web browser to access grafana UI. The URL, username and password are printed by the script 
-- In the UI, under settings -> data-sources hit `Save & test` to verify that Loki data-source is connected and that there are no errors
-- In explore tab change the data-source to `Loki` and use `{client="promtail"}` query to visualize log lines
-- Use additional queries such as `rate({client="promtail"}[1m])` to verify the behaviour of Loki and the benchmark
+### Steps
+
+1. Use the `make run-rhobs-benchmarks` or `make run-operator-benchmarks` to execute the benchmark program with the RHOBS or operator deployment styles on OpenShift respectively.
+Both commands will run all the scenarios under `config/benchmarks/scenarios/benchmarks`.
+2. Upon successful completion of each scenario, a JSON and XML file will be created in the `reports/date+time/schenario_name` directory with the results of the tests.
+3. Once all scenarios have been run we can run `python3 hack/scripts/generate_report.py $PATH_TO_SCENARIO_1 $PATH_TO_SCENARIO_2 $PATH_TO_SCENARIO_...` to compile a report that helps compare the different scenarios.
+4. To share the report on gDoc you can run `pthon3 hack/scripts/create-gdoc.py $PATH_TO_THE_REPORT` this will generate a docx file that can then be shared.
```

Comment:
```suggestion
4. To share the report on gDoc you can run `python3 hack/scripts/create-gdoc.py $PATH_TO_THE_REPORT` this will generate a docx file that can then be shared.
```